task5/servlets/Test1ForwardServlet.java

```diff
@@ -0,0 +1,24 @@
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+
+@WebServlet("/test1")
+public class Test1ForwardServlet extends HttpServlet {
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws IOException, ServletException {
+        String path = "/status";
```

Comment:
Please take on the notion of constant. Constants are invariable strings, or numbers, etc. that are just "they are as they are". Constants are declared at the beginning of the file like `private static final String PATH = "/status";`